py/circuitpy_mpconfig.h

```diff
@@ -886,7 +877,7 @@ extern const struct _mp_obj_module_t msgpack_module;
     TIME_MODULE_ALT_NAME \
 
 // This is an inclusive list that should correspond to the CIRCUITPY_XXX list above,
-// including dependencies such as TERMINALIO depending on DISPLAYIO (shown by indentation).
+// including dependencies (shown by indentation).
```

Comment:
while you're here, mind removing "(shown by indentation)"?  This is not accurate anymore, as the file is automatically indented and any manual indentation is removed.

Reply:
:+1: Sure thing.